compute_tools/src/pg_helpers.rs

```diff
@@ -214,7 +214,7 @@ impl Escaping for PgIdent {
         // Find the first suitable tag that is not present in the string.
         // Postgres' max role/DB name length is 63 bytes, so even in the
```

Comment:
It is not quite true. Yes, name can not be larger than 63 bytes. But it can be escaped in arbitrary long tag, i.e.
`$xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx$y$xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx$`

Reply:
Yeah, what I meant here is that the worst case is role/db name of `x * 63`, and it will be escaped with `x * 64` tag, which is not that bad. Since tag could be of an arbitrary length, indeed, that's not a problem